### PR TITLE
TODOs update: Notebook - load_images_tf.data.ipynb

### PR DESCRIPTION
**Pull request by siddharthchaurasia-tudip**
- Added the lab TODOs in the target notebook as needed.
- Updated the instructions in the target and solution notebook as needed.

### CODE REVIEW

File: courses/machine_learning/deepdive2/introduction_to_tensorflow/solutions/load_images_tf.data.ipynb [hunk 1]
```diff
@@ -21,7 +21,7 @@
     "In this notebook, we load an image dataset using tf.data.  The dataset used in this example is distributed as directories of images, with one class of image per directory.\n",
```
Comment (gstripling) [inline courses/machine_learning/deepdive2/introduction_to_tensorflow/solutions/load_images_tf.data.ipynb:21]: <p>Siddharth - Thanks for catching this!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1090/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (gstripling) [inline courses/machine_learning/deepdive2/introduction_to_tensorflow/solutions/load_images_tf.data.ipynb:21]: <p>Nice code addition!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1090/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (gstripling) [inline courses/machine_learning/deepdive2/introduction_to_tensorflow/solutions/load_images_tf.data.ipynb:21]: <p>Well done - using 1a and 1b!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1090/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Comment (gstripling) [inline courses/machine_learning/deepdive2/introduction_to_tensorflow/solutions/load_images_tf.data.ipynb:21]: <p>Thanks for catching this.  Please remove my comment.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1090/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

File: courses/machine_learning/deepdive2/introduction_to_tensorflow/labs/load_images_tf.data.ipynb [hunk 2]
```diff
@@ -21,7 +21,7 @@
     "In this notebook, we load an image dataset using tf.data.  The dataset used in this example is distributed as directories of images, with one class of image per directory.\n",
```
Comment (gstripling) [inline courses/machine_learning/deepdive2/introduction_to_tensorflow/labs/load_images_tf.data.ipynb:21]: <p>Thanks for seeing this and correcting it!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1090/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (gstripling) [inline courses/machine_learning/deepdive2/introduction_to_tensorflow/labs/load_images_tf.data.ipynb:21]: <p>NIcely done -- by adding the Lab Task.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1090/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Comment (gstripling) [inline courses/machine_learning/deepdive2/introduction_to_tensorflow/labs/load_images_tf.data.ipynb:21]: <p>Please remove my note comment.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1090/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Comment (gstripling) [inline courses/machine_learning/deepdive2/introduction_to_tensorflow/labs/load_images_tf.data.ipynb:21]: <p>Excellent addition!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1090/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

File: courses/machine_learning/deepdive2/introduction_to_tensorflow/solutions/load_images_tf.data.ipynb (hunk 1 above)
Comment (siddharthchaurasia-tudip) [reply courses/machine_learning/deepdive2/introduction_to_tensorflow/solutions/load_images_tf.data.ipynb:21]: Hi @gstripling , removed the comment as needed.
Thanks!

File: courses/machine_learning/deepdive2/introduction_to_tensorflow/labs/load_images_tf.data.ipynb (hunk 2 above)
Comment (siddharthchaurasia-tudip) [reply courses/machine_learning/deepdive2/introduction_to_tensorflow/labs/load_images_tf.data.ipynb:21]: Hi @gstripling , removed the comment from the cell as needed.
Thanks!

Comment (siddharthchaurasia-tudip) [reply courses/machine_learning/deepdive2/introduction_to_tensorflow/labs/load_images_tf.data.ipynb:21]: Thanks! @gstripling

**Pull request merged**